src/controllers/midi/midicontroller.cpp

```diff
@@ -160,53 +160,78 @@ void MidiController::destroyOutputHandlers() {
     }
 }
 
-QString formatMidiMessage(const QString& controllerName,
-                          unsigned char status, unsigned char control,
-                          unsigned char value, unsigned char channel,
-                          unsigned char opCode, mixxx::Duration timestamp) {
+QString MidiController::formatMidiMessage(const QString& controllerName,
```

Comment:
It wasn't in the header. It was only defined in the .cpp file, but I had to make it a private member of the class to reuse it in PortMidiController::sendShortMsg() and Hss1394Controller::sendShortMsg().